src/Compilers/VisualBasic/Portable/Binding/Binder_Expressions.vb

```diff
@@ -1242,6 +1242,28 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                 Dim enclosed = MakeRValue(memberAccess.MemberAccess, diagnostics)
                 Return memberAccess.Update(enclosed)
 
+            ElseIf expr.Kind = BoundKind.TupleLiteral AndAlso expr.Type IsNot Nothing Then
+                Dim tupleLiteral = DirectCast(expr, BoundTupleLiteral)
+                Dim args = tupleLiteral.Arguments
+                Dim rvalues As ArrayBuilder(Of BoundExpression) = Nothing
+
+                For i As Integer = 0 To args.Length - 1
+                    Dim arg = args(i)
+                    Dim rvalue = MakeRValue(arg, diagnostics)
```

Comment:
>Dim rvalue = MakeRValue(arg, diagnostics) [](start = 20, length = 41)

It feels to me that we shouldn't need to do this for properties here, it should be done when we are binding elements.